FalmouthFalcons/iteration-3
# File name conventions for PRODUCTS. Create spots for new functionality

Description:
-File name conventions for PRODUCTS. Create spots for new functionality
-Started commenting functioning code in various spots. 

Number of Fixes:
-Fixed naming conventions for clarity .

Related Ticket(s):
#4 #7 #8 #14 #2 

Problem to Solve:
- none, functionally .

Proposed Changes:
- need to correct the usage of "ORDER/S" as a :to in routing, in order to correct error and use. 

Steps to Test Solution
1) git checkout master, git pull origin master
2) git fetch 
3) git checkout adam-final

Documentation
[n  ] There is new documentation in this pull request that must be reviewed..
[ n ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
